### PR TITLE
fix typo in readme

### PR DESCRIPTION
**Pull request by cyzanfar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Add one of the following to your application.js manifest:
 
 ```js
 
-// Twitter typeahead exmaple.
+// Twitter typeahead example.
```
Comment (sjaveed) [inline README.md]: Subtle :-)

Comment (cyzanfar) [inline README.md]: very ;)

**Comment by yourabi** [comment]
Good catch. Thank you.

**Comment by cyzanfar** [comment]
Anytime :)

**Pull request merged**